pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -107,6 +106,9 @@ Follow the official [Install Docker Compose](https://docs.docker.com/compose/ins
 
 ---
 
+NOTE: If you use docker toolbox, it might downgrade your virtual box, make sure that your virtual box is up to
```

Comment:
Also move this to windows section as it is only relevant to windows platform

Reply:
line 44 is technically including the mac section as well, should I put it above the mac section?